src/secret.rs

```diff
@@ -28,7 +28,7 @@ macro_rules! impl_display_secret {
                 hasher.write(&self.secret_bytes());
                 let hash = hasher.finish();
 
-                f.debug_tuple(stringify!($thing)).field(&format_args!("#{:016x}", hash)).finish()
+                f.debug_tuple(stringify!($thing)).field(&format_args!("#{:.16x}", hash)).finish()
```

Comment:
This removes zero-padding since the type is just `u64`. Also the commit message is wrong - the type returned from `finish` is `u64`, not `Hasher`. We should just drop this commit.

Reply:
No sweat, dropped.